lib/app/frontend/rapidDevServer/RapidApi.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const request = require('request')
+const appStartCommand = require('./appStart.json')
+
+class RapidApi {
+  /**
+   * Set the settings necessary to send requests to the RAPID.
+   * @param {Object} options Settings for requesting the RAPID
+   * @param {string} options.url Endpoint of the RAPID that processes the http requests
+   * @param {string} options.shopNumber Number of the shop for which the request should be made
+   * @param {string} options.sessionId SessionId from cake (required e.g. for getting prices)
+   * @param {string} options.deviceId Device identifier necessary for pipelines to access storages
+   */
+  constructor (url, shopNumber) {
+    this.settings = { url, shopNumber }
+    this.handle = this.handle.bind(this)
+  }
+
+  init (callback = () => {}) {
+    this.requestIds((error, sessionId, deviceId) => {
+      if (error) {
+        throw error
+      }
+
+      this.setIds(sessionId, deviceId)
+
+      console.log(`    STACK: ${this.settings.url}`)
```

Comment:
replace console.log with our logger instead, this looks like "logger.info()" to me